res/controllers/Pioneer-DDJ-400-script.js

```diff
@@ -44,7 +44,7 @@
 //        * Keyshift mode
 
 var PioneerDDJ400 = {};
-
+engine.setValue("[EffectRack1_EffectUnit1]", "focused_effect", 3);
```

Comment:
I think this slipped in by accident?
(it's in the wrong place, and also looks like a personal preference since the focused effect is stored in mixxx.cfg)
Please revert.

Reply:
No, it was like that before, it makes sure that there is an effect selected on startup.

Reply:
This line didn't exist, that's way it shows up in the diff. Also, it's in the wrong place.
Please revert.